tests/e2e/testcases/blockchain/deploy/suite.go

```diff
@@ -246,6 +261,31 @@ var _ = ginkgo.Describe("[Blockchain Deploy]", ginkgo.Ordered, func() {
 		gomega.Expect(len(localClusterUris)).Should(gomega.Equal(1))
 	})
 
+	ginkgo.It("HAPPY PATH: local deploy with signature aggregator endpoint set", func() {
```

Comment:
can you add a path where a signature aggregator is locally started but the endpoint is not given? does deploy use the previously started signature aggregator?